packages/block-editor/src/components/url-input/index.js

```diff
@@ -256,7 +251,7 @@ class URLInput extends Component {
 			id,
 			isFullWidth,
 			hasBorder,
-			renderSuggestions,
+			__experimentalRenderSuggestions,
```

Comment:
Could potentially use an alias on this and other props, as its really only the public interface that needs to be made experimental:
```
__experimentalRenderSuggestions: renderSuggestions
```

 but I think it's ok either way. I'll leave it up to you.

Reply:
Fixed as it makes things easier to read and will avoid merge conflicts.